components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -16,6 +16,8 @@
 
 #import "MaterialShadowElevations.h"
 
+extern CGFloat const kPresentedViewCorenerRadius;
```

Comment:
nit: Corner

Consider removing the k as this is not a constant from CoreFoundation.  (I know we are not consistent on this)

Reply:
Also naming: s/Corener/Corner/

Reply:
Since this is a public global constant, please start with the component prefix, so something like `MDCDialogsPresentedViewCornerRadius`